src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -264,6 +268,17 @@ const ActionsPage = () => {
 
   return (
     <div className={styles.main}>
+      <div className={styles.container}>
+        <p className={styles.recoveryTag}>
+          <FormattedMessage {...MSG.recoveryTag} />
+        </p>
+      </div>
+      {actionType === ColonyActions.Recovery && (
```

Comment:
I think this is old code right? It looks like the old tag before the improvement you did, the conditional should be wrapped around the div above this.

Reply:
Yes it is old code. Pull the new version, and then re-review it, as It's all good now.